types/Config.js

```diff
@@ -61,7 +61,7 @@ export type DefaultOptions = {|
   watchman: boolean,
 |};
 
-export type InitialOptions = {|
```

Comment:
this object is not supposed to be exact. It comes from the config file that can have anything in it. We guard against unrecognized options in `jest-validate` package.